### PR TITLE
Update UML diagrams with University

### PR DESCRIPTION
**Pull request by verakohh**
due:Thursday

### CODE REVIEW

File: docs/diagrams/ModelClassDiagram.puml [hunk 1]
```diff
@@ -4,23 +4,24 @@ skinparam arrowThickness 1.1
...
-Class UniquePersonList
-Class Person
-Class Address
-Class Email
-Class Name
-Class Phone
-Class Tag
-
-Class I #FFFFFF
+Package Model as ModelPackage <<Rectangle>> {
```
Comment (chenle228) [inline docs/diagrams/ModelClassDiagram.puml:7]: The refactoring here is very clean and clear. Adding the University class is a valuable update, and the logical placement in the existing structure is intuitive. Great job on following the existing design structure.

File: src/test/java/seedu/address/logic/commands/FindByWorkExperienceCommandTest.java [hunk 2]
```diff
@@ -71,11 +67,13 @@ public void execute_zeroKeywords_noPersonFound() {
     @Test
     public void execute_multipleKeywords_multiplePersonsFound() {
         String expectedMessage = String.format(MESSAGE_PERSONS_LISTED_OVERVIEW, 2);
-        WorkExperienceContainsKeywordsPredicate predicate = new WorkExperienceContainsKeywordsPredicate("Intern", "Google", "2024");
+        WorkExperienceContainsKeywordsPredicate predicate =
```
Comment (chenle228) [inline src/test/java/seedu/address/logic/commands/FindByWorkExperienceCommandTest.java:70]: The modification of WorkExperienceContainsKeywordsPredicate is a solid update that better reflects the use case by including the additional keywords Intern, Google, 2024. This makes the search functionality more flexible. Ensure that the keywords being tested align with the team’s intended use cases for future enhancements or additional edge cases.

File: src/test/java/seedu/address/logic/parser/FindByWorkExperienceCommandParserTest.java [hunk 3]
```diff
@@ -16,7 +16,8 @@ public class FindByWorkExperienceCommandParserTest {
     @Test
     public void parse_validArgs_returnsFindCommand() {
         FindByWorkExperienceCommand expectedCommand =
-                new FindByWorkExperienceCommand(new WorkExperienceContainsKeywordsPredicate("Intern", "Google", "2024"));
+                new FindByWorkExperienceCommand(new WorkExperienceContainsKeywordsPredicate("Intern",
```
Comment (chenle228) [inline src/test/java/seedu/address/logic/parser/FindByWorkExperienceCommandParserTest.java:19]: The inclusion of "2024" within the command parser is a great improvement for specificity. However, I recommend adding a few more test cases to ensure that invalid or partial years are caught by this parser, for instance, "202", "24", or non-numerical strings.

File: src/test/java/seedu/address/logic/parser/FindByWorkExperienceCommandParserTest.java [hunk 4]
```diff
@@ -32,4 +33,4 @@ public void parse_invalidArgs_throwsParseException() {
         assertParseFailure(parser, " ", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
                 FindByWorkExperienceCommand.MESSAGE_USAGE));
     }
-}
\ No newline at end of file
+}
```
Comment (chenle228) [inline src/test/java/seedu/address/logic/parser/FindByWorkExperienceCommandParserTest.java:36]: Excellent work on keeping the method structure clean and readable. It’s easy to follow the logic.

File: README.md [hunk 5]
```diff
@@ -38,4 +38,4 @@ UniVerse is a **comprehensive platform** tailored for university students to sea
 
 UniVerse provides university students with a **streamlined solution to manage their personal and professional networks**, enabling them to grow both socially and academically in one unified platform.
 
-* This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org). For the detailed documentation of the project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**
\ No newline at end of file
+* This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org). For the detailed documentation of the project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**
```
Comment (chenle228) [inline README.md:41]: The addition of the "AddressBook-Level3 project" information is a good touch.

**Review by chenle228 — COMMENTED** [review]
LGTM

File: src/test/java/seedu/address/logic/commands/FindByWorkExperienceCommandTest.java (hunk 2 above)
Comment (jeremychiaaaaa) [reply src/test/java/seedu/address/logic/commands/FindByWorkExperienceCommandTest.java:70]: The update to WorkExperienceContainsKeywordsPredicate is a good improvement, as it now supports additional keywords like "Intern," "Google," and "2024," which enhances the flexibility of the search functionality. Just make sure that the tested keywords are in line with the team's intended use cases, especially for any future updates or edge cases.

**Review by jeremychiaaaaa — APPROVED** [review]
LGTM
